commands/narrator/rematch.js

```diff
@@ -1,33 +1,84 @@
+
+            await message.guild.channels.cache
+                .find((x) => x.name == "day-chat")
+                .permissionOverwrites.edit(ids.alive, {
+                    SEND_MESSAGES: false,
+                    READ_MESSAGE_HISTORY: false,
+                    VIEW_CHANNEL: false,
+                })
+
+            await message.guild.channels.cache
+                .find((c) => c.name === "vote-chat")
+                .permissionOverwrites.edit(ids.alive, {
+                    SEND_MESSAGES: false,
+                    READ_MESSAGE_HISTORY: true,
+                    VIEW_CHANNEL: true,
+                })
+
+            await message.guild.channels.cache
+                .find((x) => x.name == "game-lobby")
+                .permissionOverwrites.edit(ids.alive, {
+                    SEND_MESSAGES: true,
+                    READ_MESSAGE_HISTORY: true,
+                    VIEW_CHANNEL: true,
+                })
+
+            db.set(`gamePhase`, -10)
+
+            let wwChat = message.guild.channels.cache.find((c) => c.name === "werewolves-chat")
+            let zombiesChat = message.guild.channels.cache.find((c) => c.name === "zombies-chat")
+            let siblingsChat = message.guild.channels.cache.find((c) => c.name === "siblings-chat")
+            let wwVote = message.guild.channels.cache.find((c) => c.name === "ww-vote")
+
+            db.get(`players`)?.forEach(async (p) => {
+                await wwChat.permissionOverwrites.delete(p)
+                await wwVote.permissionOverwrites.delete(p)
+                await zombiesChat.permissionOverwrites.delete(p)
+                await siblingsChat.permissionOverwrites.delete(p)
+
+                let member = await message.guild.members.fetch(p).catch((a) => null)
+                await member?.roles?.set(member.roles.cache.filter((r) => !["Corrupted", "Revealed"].includes(r.name)).map((r) => (r.name === "Dead" ? ids.alive : r.id)))
+            })
+
+            db.delete(`players`)
+
+            db.all()
+                .filter((data) => data.ID.startsWith("player_"))
+                .forEach((data) => db.delete(data.ID))
+
+            let save = ["rankedseason", "stafflist", "stafflist2", "stafflist3", "entermsg", "hoster", "gamePhase", "started", "usedChannels", "wwsVote", "winner", "vtshadow", "xpGiven", "nextRequest", "gamewarnIndex", "gamewarnCount", "logs", "gameCode", "game", "maintance", "xpExclude"]
+
+            db.all().forEach(async (i) => {
+                if (i.ID.startsWith("roses_")) {
+                    await players.updateOne({ user: i.ID.replace("roses_", "") }, { $inc: { roses: 1 } })
                 }
-            }
-        })
+                if (!save.includes(i.ID)) db.delete(i.ID)
+            })
+            const temproles = message.guild.channels.cache.find((x) => x.name == "private channels")
+            await temproles?.children?.forEach(async (channel) => {
+                await channel.delete()
+            })
```

Comment:
Instead of doing all this you could also just run the reset command. And tbh I don't know why all that stuff instead of the stuff I wrote xD

Reply:
The reset command will delete the `game.id` object which is needed to disable buttons. The long code is to remove all permissions from players in channels like werewolves-chat and zombies-chat to avoid players seeing those channels when the rematch has started